app/assets/javascripts/backbone/views/application_view.coffee

```diff
@@ -30,3 +32,19 @@ class MS.Views.ApplicationView extends Backbone.View
         self.$('.ms-popover').popover('hide')
         self.$el.off 'click', hide # Unregister for performance reasons
     @$el.on 'click', hide
+
+  showErrorModal: (error = MS.errorModalDefaultText) ->
+    MS.errorModal.find('.modal-body').text(error)
+    MS.errorModal.modal('show')
+
+  handleAjaxErrors: ->
+    $(document)
+      .ajaxError (e, jqXHR) =>
+        e.stopPropagation()
```

Comment:
Not sure if this is necessary since we're listening on `document` but I guess it couldn't hurt.